include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -859,6 +859,50 @@ class MGTransferGlobalCoarsening : public dealii::MGTransferBase<VectorType>
   build(const std::function<void(const unsigned int, VectorType &)>
           &initialize_dof_vector);
 
+  /**
+   * Default constructor.
```

Comment:
Can you document how one initializes the class if using the default constructor? How do you pass ``transfer`` and ``initialize_dof_vector`` then?

Reply:
This is still work in progress. I needed to add this constructor because `MGTransferMatrixFree` has this.